logrotate/test/FilterTest.h

```diff
@@ -10,7 +10,9 @@
 
 #pragma once
 #include <gtest/gtest.h>
+#ifdef __linux__ || __APPLE__  || __unix__
```

Comment:
This fails to compile in OSX

<img width="609" alt="screen shot 2016-05-12 at 2 06 04 am" src="https://cloud.githubusercontent.com/assets/331742/15207890/23d8b23e-17e6-11e6-944a-12a9cef3eda2.png">